vega_sim/scenario/comprehensive_market/agents.py

```diff
@@ -0,0 +1,25 @@
+from collections import namedtuple
+
+WalletConfig = namedtuple("WalletConfig", ["name", "passphrase"])
+
+# Set up wallets for parties proposing market
+MM_WALLET = WalletConfig("mm", "pin")
+
+# Set up wallets for parties submitting market orders
+lp_agents = 5
+LP_WALLETS = [WalletConfig("lp_trader_" + str(i), "pass") for i in range(lp_agents)]
+
+# Set up wallets for parties submitting market orders
+mo_agents = 5
+MO_WALLETS = [WalletConfig("mo_trader_" + str(i), "pass") for i in range(mo_agents)]
+
+# Set up wallets for parties submitting / cancelling limit orders
+lo_agents = 20
+LO_WALLETS = [WalletConfig("lo_trader_" + str(i), "pass") for i in range(lo_agents)]
```

Comment:
Could we make all of these configurable in the scenario itself rather than here? (i.e. as a `num_market_order_agents` parameter)